src/ui/components/LedgerViewer.js

```diff
@@ -51,6 +58,13 @@ const useStyles = makeStyles((theme) => {
   };
 });
 
+const DATE_SORT = deepFreeze({
+  name: Symbol("Date"),
+  fn: (e: LedgerEvent) => e.ledgerTimestamp,
+});
+
+const PAGINATION_OPTIONS = deepFreeze([25, 50, 100]);
```

Comment:
Curious to hear why we are deepFreezing these consts.

Reply:
Dandelion recommended it. It seems to align with our pattern of having the most restrictive writability possible for stored values.